todo-backend/todo-backend.yaml

```diff
@@ -0,0 +1,43 @@
+# This configuration file can be used to build and deploy the todo-backend
+# quickstart on OpenShift with the Helm Chart for WildFly.
+build:
+  uri: https://github.com/jmesnil/quickstart.git
+  ref: todo-backend
+  mode: bootable-jar
+  env:
+    - name: ARTIFACT_DIR
+      value: todo-backend/target
+    - name: MAVEN_ARGS_APPEND
+      value: -am -pl todo-backend -P bootable-jar-openshift -Dversion.server.bom=23.0.0.Final -Dversion.microprofile.bom=23.0.0.Final
```

Comment:
We need to get rid of the versions or somehow parameterize them

Reply:
well... this is silly.

We can not build the image from the quickstart master branch as it relies on SNAPSHOTS of WildFly.
So if I want to build the image, I must pass Final release for WildFly and BOMS.

Now, once we have a tag with the quickstart (that uses Final release of WildFly), we will not need to override them... but we will have to update this YAML for every new release of the quickstart...

Reply:
I understand, for testing purposes this was ok (and really one of the few ways how to actually test it), I just noted that because this should not be there in the time of merging

Reply:
you were right: let's have the PR being "clean" and do local changes for testing